Azure/azure-ad/_meta/fields.yml

```
@@ -275,6 +275,11 @@ azuread.resultSignature:
   name: azuread.resultSignature
   type: keyword
 
+azuread.scopes:
+  description: ''
+  name: azuread.scopes
+  type: array
+
```

As the data came from the field `properties` in the event, I propose to rename the field accordingly:
```suggestion
azuread.properties.scopes:
  description: ''
  name: azuread.properties.scopes
  type: array

```